nodejs/user-interface/modals/new-task.js

```diff
@@ -1,8 +1,6 @@
 const { Modal, Blocks, Elements } = require('slack-block-builder');
-// Need to import user id to set as `initialUser` or `initialConversation`
-//const { User} = require('../models'); 
 
-module.exports = (prefilledTitle) => {
+module.exports = (prefilledTitle, currentUser) => {
```

Comment:
Rather than trying to pull logic around querying users, it's better in this view layer to focus on how the information is presented, so what you can do is add a new argument to the method that requires the userID be passed, ready for the view code to pass it into the block